packages/claims/src/claims/claims.ts

```diff
@@ -70,20 +73,30 @@ export class Claims implements IClaims {
    * @param hashFns {{ [alg: string]: (data: string) => string }}
    */
   async verify(
-    claimUrl: string, hashFns?: { [alg: string]: (data: string) => string },
+    claimUrl: string,
+    hashFns?: { [alg: string]: (data: string) => string },
   ): Promise<IPublicClaim | IPrivateClaim> {
     const token = await this.store.get(claimUrl);
-    const claim = this.jwt.decode(token) as
-      (IPublicClaim | IPrivateClaim) & { iss: string; sub: string };
+    const claim = this.jwt.decode(token) as (IPublicClaim | IPrivateClaim) & {
+      iss: string;
+      sub: string;
+    };
     if (!(await this.verifySignature(token, claim.iss))) {
       throw new Error('Invalid signature');
     }
-    if (!this.document.isValidDelegate(DelegateTypes.verification, claim.signer, claim.did)) {
-      throw new Error('Issuer isn\'t a use\'r delegate');
+    if (
+      !this.document.isValidDelegate(
```

Comment:
In my mind, part of this story is to consolidate the verification code that is in this `verify` function with the `verify` function that is in `AuthTokenVerifier`. For instance, a crucial difference is that `AuthTokenVerifier.verify` allows for either `isIdentity` or `isDelegate` (i.e. `(await this.isIdentity(token)) || (await this.isDelegate(token))`), while this code is just `isDelegate` at the moment

Reply:
@jrhender It makes sense to authenticate issuer in verification methods of `claims` package by `AuthTokenVerifier`. It will cause reading of document thought. Is this acceptable?